doc/usecases/plugins/spec/create_specification.md

```diff
@@ -0,0 +1,55 @@
+# Use Case: Writing specification for an application
+
+## Summary
+
+- **Scope:** `spec`
+- **Level:** Developer Goal
+- **Actors:** Application, User
+- **Brief:**
+  - Creating a specification for a mail client application which needs communication to a mail server.
+
+## Scenarios
+
+- **Precondition:** kdb is installed.
+- **Main success scenario:**
+  - Specification for mail server application is available.
```

Comment:
This would be a precondition. But I think the use case should be about writing, i.e., doing several meta-set of the specification (so nothing needs to be available beforehand)

Reply:
Thanks!